programs/zstdcli.c

```diff
@@ -1213,6 +1227,23 @@ int main(int const argCount, const char* argv[])
         if (adaptMin > cLevel) cLevel = adaptMin;
         if (adaptMax < cLevel) cLevel = adaptMax;
 
+        if (showDefaultCParams) {
+            size_t fileNb;
+            for (fileNb = 0; fileNb < (size_t)filenames->tableSize; fileNb++) {
+                const size_t fileSize = UTIL_isRegularFile(filenames->fileNames[fileNb]) ? (size_t)UTIL_getFileSize(filenames->fileNames[fileNb]) : 0;
```

Comment:
The casting to `size_t` doesn't seem useful.
It seems to keep the information in its original format, `unsigned long long`.

Reply:
Whoops. sounds good!